src/main/java/jenkins/plugins/openstack/compute/internal/Openstack.java

```diff
@@ -521,7 +520,7 @@ private boolean isOurs(@Nonnull Server server) {
                 }
                 throw err;
             }
-            debug("Machine started: " + server.getName());
+            debug("Machine started: {0}", server.getName());
```

Comment:
This kind of change allows the code to avoid creating a fully formatted String until _after_ it's decided that this log level is enabled.